src/current/_includes/v24.1/prod-deployment/process-termination.md

```diff
@@ -1,5 +1,9 @@
 {{site.data.alerts.callout_danger}}
-We do not recommend sending `SIGKILL` to perform a "hard" shutdown, which bypasses CockroachDB's [node shutdown logic](#node-shutdown-sequence) and forcibly terminates the process. This can corrupt log files and, in certain edge cases, can result in temporary data unavailability, latency spikes, uncertainty errors, ambiguous commit errors, or query timeouts. When decommissioning, a hard shutdown will leave ranges under-replicated and vulnerable to another node failure, causing [loss of quorum]({% link {{ page.version.version }}/architecture/replication-layer.md %}#overview) in the window before up-replication completes.
+Cockroach Labs does not recommend sending a `SIGKILL` signal to the `cockroach` process on a node to forcibly terminate it, because it bypasses CockroachDB's [node shutdown logic](#node-shutdown-sequence) and degrades the cluster's health. From the point of view of other cluster nodes, the node will be suddenly unavailable.
```

Comment:
nit: Technically you can't send a SIGKILL to the process. You send a SIGKILL to Linux and it terminates the process ungracefully.  This is clear enough that you don't have to change it, but if you wanted to make it slightly more accurate, you could write:

> Cockroach Labs does not recommend terminating the `cockroach` process with a `SIGKILL` signal, because ...